proposals/0024-opacity-micromaps.md

```diff
@@ -200,46 +217,79 @@ In addition to the AST traversal detecting any explicit use of the new flag,
 the same DefaultError warning diagnostic will be added to detect when the
 new ray flag is used at the `RayQuery` template argument, `TraceRay()`, or
 `RayQuery::TraceRayInline()` (when it is immediate).
```

Comment:
The preceding paragraph talks about checking immediate constant RayFlags values on the method calls.  We decided not to do this and only use the availability check mechanism.

Two paragraphs earlier (I can't comment directly on unchanged areas), it talks about looking for the `RaytracingPipelineFlags::AllowOpacityMicromaps` in the flags for the Raytracing pipeline config1 subobject.  This is on top of checking availability attributes on expressions used.  We should remove this paragraph as well, since we decided not to add this checking.